changelog.md

```diff
@@ -54,13 +54,18 @@ It is possible to make Marathon let Mesos use the `TASK_KILLING` state introduce
 ## Fixed issues
 
 - #929 - Allow tcp,udp ports in portMappings
+- #2751 - Commit suicide on ZK exceptions
+- #3091 - App updates hanging on downscales 
 - #3169 - Possible to start app with negative resources
 - #3241 - Serverside validation messages are inconsistent
+- #3251 - Tried to kill an existing app, said it doesn't exist even though it does 
```

Comment:
This was backported to v0.15 and included in v0.15.3.

We didn't add a changelog for v0.15.3, but we probably should.